data/features/flexbox-gap.js

```diff
@@ -8,11 +8,21 @@
  * @type {import('../features').Feature}
  */
 export default (rule) => {
-  if (/display:\s(inline-)?flex/.test(rule.toString())
-    && rule.toString().includes('gap:')
-  ) {
-    return true;
+  if (!('some' in rule)) {
+    return false;
   }
 
-  return false;
+  let isFlexbox = false;
```

Comment:
`hasFlexbox` is probably clearer since we're going to look at multiple declarations and see if one matches the check. (same with `hasGap`).